apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,150 @@ const data: FoundAnimalFlow = {
+        prompt: [
+          "Does the cat appear to be sick, injured, in danger, or a nursing kitten with no mama in sight?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "If the cat appears to be sick, injured, in danger, or if you've found a nursing kitten with no mama in sight please contact an animal welfare organization or shelter that can help you assess the situation",
+              ],
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Many cats that people spot in neighborhoods aren't in need of immediate help. Instead, they are pets who spend some or all of their time outdoors.",
+                "Check for a collar, if the cat has one try and get in contact with the owner, If the cat has no collar you can take the cat to the nearest animal shelter to check for a microchip",
+                "If the cat does not have a microchip, assess whether the cat seems healthy (not injured or visibly ill) and safe (in a secure location, away from any danger) We want a healthy and safe cat to stay where it is while you attempt to find the owner",
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Raccoon",
+      flow: {
+        prompt: [
+          "Does the raccoon appear to be sick or injured?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "Is the raccoon an adult or a baby?",
+              ],
+              options: [
+                {
+                  name: "Adult",
+                  flow: {
+                    prompt: [
+                      "If you have found an injured adult raccoon, although it may be hard to accept. Adult raccoons can almost never be caught and successfully treated. If you see an adult raccoon with an injury, leave the animal alone. Even though the injury may take a long time to heal, this is far preferable to the trauma of chase and capture.",
+                    ],
+                  },
+                },
+                {
+                  name: "Baby",
+                  flow: {
+                    prompt: macros.general.rehab("the baby raccoon"),
+                  },
+                },
+              ],
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Have you found a baby raccoon that's alone with no mother in sight?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "Be very careful not create a orphan raccoon on accident. When a baby raccoon is separated from its mother, the baby will stay where it is until the mother returns. It's best to observe the baby without disturbing it.",
```

Comment:
```suggestion
                      "Be careful not to create an orphan raccoon accidentally. When a baby raccoon is separated from its mother, it will stay where it is until the mother returns.",
                      "Monitor the baby from a distance to make sure the mother is still caring for it. Do not attempt to feed or otherwise care for the baby, as this may lead to it imprinting on humans and becoming dependent on them.",
```